pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter_test.go

```diff
@@ -141,7 +140,7 @@ var _ = Describe("Validating VMIUpdate Admitter", func() {
 			)
 
 			DescribeTable("and NodeName set", func(handlernode string, matcher types.GomegaMatcher) {
-				vmi := api.NewMinimalVMI("testvmi")
+				vmi := libvmi.New(libvmi.WithName("testvmi"))
```

Comment:
Can you please remove the `libvmi.WithName()` part?
The name is not important here, and libvmi will just create a random one
```suggestion
				vmi := libvmi.New()
```

Reply:
Oh, I see! I’ve removed it. Thanks for your feedback!